x/ccv/consumer/types/keys.go

```diff
@@ -65,8 +65,9 @@ const (
 	// [DEPRECATED]
 	LastStandaloneHeightByteKey
 
-	// SmallestNonOptOutPowerByteKey is the byte that will store the smallest val power that cannot opt out
-	SmallestNonOptOutPowerByteKey
+	// NOTE: This key is deprecated, but left in place to avoid consumer state migrations
+	// [DEPRECATED]
+	DeprecatedSmallestNonOptOutPowerByteKey
```

Comment:
I like the rename, we didn't do this for other deprecated keys; not sure what the implications are, but I think it's ok to have it like this.

Reply:
The renaming should not have an impact on anything. We just leave `DeprecatedSmallestNonOptOutPowerByteKey` there, so the `byte` value of it (i.e., `10`) is not reused by some other key/prefix, because otherwise, this would break the store.